content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,200 @@
+   enable specific tasks.
+
+*  **Portable:** A configuration bundle for a complex containerized workload
+   might include service account definitions for the system's components. The
+   lightweight nature of service accounts and the namespaced identities make
+   the configurations portable.
+
+Service accounts are different from user accounts, which are authenticated
+human users in the cluster. User accounts don't exist in the Kubernetes API
+server; instead, the API server treats user identities as opaque data. You can
+authenticate as a user account using multiple methods.
+
+{{< table caption="Comparison between service accounts and users" >}}
+
+| Description | ServiceAccount | User or group |
+| --- | --- | --- |
+| Location | Kubernetes API (ServiceAccount object) | External |
+| Scope | Namespace | Cluster |
+| Access control | Kubernetes RBAC or other [authorization mechanisms](/docs/reference/access-authn-authz/authorization/#authorization-modules) | Kubernetes RBAC or other identity and access management mechanisms |
+| Intended use | Pods | People |
+
+{{< /table >}}
+
+### Default service accounts
+
+When you create a cluster, Kubernetes automatically creates a ServiceAccount
+object named `default` for every namespace in your cluster. The `default`
+service accounts in each namespace get no permissions other than the
+[default API discovery permissions](/docs/reference/access-authn-authz/rbac/#service-account-permissions)
+that Kubernetes grants to all authenticated accounts. If you delete the default
+ServiceAccount object in a namespace, the {{<glossary_tooltip text="control plane" term_id="control-plane">}}
+replaces it with a new one.
+
+If you deploy a Pod in a namespace, and you don't manually assign a
+ServiceAccount object to the Pod, Kubernetes assigns the `default` service
+account for that namespace to the Pod. You can change this behavior by setting
+the `automountServiceAccountToken` field in your Pod specification to `false`.
+
+## Use cases for Kubernetes service accounts
+
+As a general guideline, you can use service accounts to provide identities in
+the following scenarios:
+
+* Your Pods need to communicate with the Kubernetes API server, for example in
+  situations such as the following:
+  *  Providing read-only access to sensitive information stored in Secrets.
+  *  Granting [cross-namespace access](TODO), such as allowing a Pod in namespace
+     `example` to read, list, and watch for Lease objects in the
+     `kube-node-lease` namespace.
+* Your Pods need to communicate with an external service. For example,
+  [Authenticating to a private image registry using an `imagePullSecret`](/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account).
+* An external service needs to communicate with the Kubernetes API server. For
+  example, authenticating to the cluster as part of a CI/CD pipeline.
```

Comment:
Is it better if we have some references for the external service use cases? we can make a section of it such as `Use ServiceAccount credentials from external`. It will be better to understand the things

Reply:
We can do that, but from what I see right now we only have one defined use case. Maybe after we publish this you'd be willing to open a new issue or PR with more to add?

Reply:
I would like to know that use case, could you please refer that link to me.

Reply:
Are you talking about **authenticating to the cluster as part of a CI/CD pipeline**?

Reply:
Yeah @Shubham82 that's the only example I have for this PR. We can always expand with more examples down the line though

Reply:
> We can always expand with more examples down the line though

Sure
Thanks, @shannonxtreme for this.